src/wp-includes/widgets.php

```diff
@@ -1042,19 +1042,27 @@ function wp_get_sidebars_widgets( $deprecated = true ) {
  * @access private
  *
  * @global array $_wp_sidebars_widgets
- * @param array $sidebars_widgets Sidebar widgets and their settings.
+ * @global array $sidebars_widgets
+ * @param array $new_sidebars_widgets Sidebar widgets and their settings.
+ * @param boolean $refresh_global_sidebars_widgets Should update the global $sidebars_widgets variable?
```

Comment:
Formatting for coding standards and param naming to be consistent with other boolean param types:

```suggestion
 * @global array $sidebars_widgets
 *
 * @param array   $new_sidebars_widgets            Sidebar widgets and their settings.
 * @param boolean $refresh_global_sidebars_widgets Optional. Whether to update $sidebars_widgets
 *                                                 global. Default true.
```